drivers/hdc1000/hdc1000.c

```diff
@@ -72,6 +76,11 @@ int hdc1000_init(hdc1000_t *dev, const hdc1000_params_t *params)
     }
     i2c_release(dev->p.i2c);
 
+    /* set initial value for hdc device */
+    dev->last_read_time = 0xFFFFFFFF;
```

Comment:
Why do you initialize with `0xFFFFFFFF`? This may lead to overflows in
`if(now + last > INTERVAL)` so the statement not apply in the first run. Better initialize with `0` and adapt the `if` statement (see below)

Reply:
This value is to use for checking initial condition, not for actual outdate check.